help/upgrade/patches/overview.md

```diff
@@ -85,8 +85,8 @@ index c8a6fef58d31..7d01c195791e 100644
 You can apply patches using any of the following methods:
 
 -  [Quality Patches Tool](https://devdocs.magento.com/quality-patches/tool.html)
--  [Command line](../patches/apply.md#command-line)
--  [Composer](../patches/apply.md#composer)
+-  [Command line](https://experienceleague.adobe.com/docs/commerce-operations/upgrade-guide/patches/apply.html?lang=en#command-line)
```

Comment:
```suggestion
-  [Command line](/help/upgrade/patches/apply.md#command-line)
```

Reply:
Committed your suggestions 👍 and thanks for sharing the link syntax guidelines. The idea of setting up the docs repo on local development machine and maintain the doc is great. But as I don't actively update/maintain the docs repo, and it might be the case for most of us, I tend to go for 'Converting URLs to hyperlinks' route.

Reply:
No problem. After we merge this, please check if links work for you and report back here.